upgrades/steps124_test.go

```diff
@@ -156,6 +156,31 @@ func (s *steps124Suite) TestMoveSyslogConfig(c *gc.C) {
 
 }
 
+func (s *steps124Suite) TestMoveSyslogConfigCantDeleteOld(c *gc.C) {
+	logdir := c.MkDir()
+	datadir := c.MkDir()
+	data := []byte("data!")
```

Comment:
nit: given that this is a) obvious and b) only used once, surely you don't need a variable for it